<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameStartPageClass</class>
 <widget class="QWidget" name="GameStartPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameStartPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="joinGamePushButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">.QPushButton {
  border-radius: 4px;
  background-color: #0e6fa2;
  border: none;
  color: #000;
  text-align: center;
  font-size: 22px;
  padding: 10px;
  margin: 5px;
}
.QPushButton:hover{
  border-radius: 5px;
  background-color: #38afd1;
  margin: 3px;
  color:#FFF;
}</string>
     </property>
     <property name="text">
      <string>Join game</string>
     </property>
     <property name="iconSize">
      <size>
       <width>19</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="createGamePushButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">.QPushButton {
  border-radius: 4px;
  background-color: #0e6fa2;
  border: none;
  color: #000;
  text-align: center;
  font-size: 22px;
  padding: 10px;
  margin: 5px;
}
.QPushButton:hover{
  border-radius: 5px;
  background-color: #38afd1;
  margin: 3px;
  color:#FFF;
}</string>
     </property>
     <property name="text">
      <string>Create game</string>
     </property>
     <property name="iconSize">
      <size>
       <width>19</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="viewHistoryPushButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">.QPushButton {
  border-radius: 4px;
  background-color: #0e6fa2;
  border: none;
  color: #000;
  text-align: center;
  font-size: 22px;
  padding: 10px;
  margin: 5px;
}
.QPushButton:hover{
  border-radius: 5px;
  background-color: #38afd1;
  margin: 3px;
  color:#FFF;
}</string>
     </property>
     <property name="text">
      <string>View history</string>
     </property>
     <property name="iconSize">
      <size>
       <width>19</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="logOutPushButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">.QPushButton {
  border-radius: 4px;
  background-color: #0e6fa2;
  border: none;
  color: #000;
  text-align: center;
  font-size: 22px;
  padding: 10px;
  margin: 5px;
}
.QPushButton:hover{
  border-radius: 5px;
  background-color: #38afd1;
  margin: 3px;
  color:#FFF;
}</string>
     </property>
     <property name="text">
      <string>Log out</string>
     </property>
     <property name="iconSize">
      <size>
       <width>19</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
